src/pages/teams/[teamId]/dashboard/residents/[residentId]/document/[documentSubmissionId].tsx

```diff
@@ -237,12 +243,12 @@ const DocumentDetailPage: NextPage<WithUser<DocumentDetailPageProps>> = ({
         </figcaption>
       </figure>
 
-      {documentSubmission.rejectedAt && (
-        <div>
-          <h2 className="lbh-heading-h3">History</h2>
-          <History documentSubmission={documentSubmission} />
-        </div>
-      )}
+      {/* {documentSubmission.rejectedAt && ( */}
```

Comment:
I think we can remove this commented out block?

Reply:
👍